CHANGELOG.rst

```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+7.0.0 - 2022-07-20
```

Comment:
We also merged two PRs that support Hotel Autocomplete and Airlines Routes from the community, right?

Reply:
OMG YES Xianqi!!! Will update asap, thank you <3